tests/llm/test_completions/test_completion_create.py

```diff
@@ -0,0 +1,218 @@
+import pytest
+import instructor
+from pydantic import BaseModel
+from cohere import Client, AsyncClient
+from cerebras.cloud.sdk import Cerebras, AsyncCerebras
+from openai import OpenAI, AsyncOpenAI
+from anthropic import Anthropic, AsyncAnthropic
+from mistralai import Mistral
+from google.generativeai import GenerativeModel as GenerativeAIGenerativeModel
+import os
+from vertexai.generative_models import GenerativeModel
+from writerai import Writer, AsyncWriter
+from fireworks.client import Fireworks, AsyncFireworks
+from google.genai import Client as GoogleGenaiClient
+
+
+class User(BaseModel):
+    name: str
+    age: int
+
+
+response_model_and_prompt = [
+    (User, "Ivan is 20 and lives in Singapore?"),
+    (bool, "Is 30 greater than 20?"),
+]
+clients = [
```

Comment:
Would this work? This would mean that I have to install every single client right?

Reply:
yes this means you need to install every single client with this so we cannot run this in CI. Would you prefer just testing a subset of clients in CI and having the rest commented out?